operations/environments/prod/main.tf

```
@@ -0,0 +1,33 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.113.0"
+    }
+  }
+
+  # Use a remote Terraform state in Azure Storage
+  backend "azurerm" {
+    resource_group_name  = "cdcti-terraform"
+    storage_account_name = "cdctiterraform"
+    container_name       = "tfstate"
```

```suggestion
    container_name       = "terraform-state"
```
I think this should be `terraform-state` just like staging.